system/services/presideObjects/PresideObjectService.cfc

```diff
@@ -2189,6 +2190,53 @@ component displayName="Preside Object Service" {
 		return fields;
 	}
 
+	public array function parseSelectFieldsForHavingClause(
```

Comment:
I think the name of this function is a little confusing/misleading. Could this be changed to `expandHavingClauses()`?